omnibus/config/software/supermarket-cookbooks.rb

```diff
@@ -41,6 +41,7 @@
     open("#{cookbooks_path}/solo.rb", "w") do |file|
       file.write <<-EOH.gsub(/^ {8}/, '')
         cookbook_path   "#{cookbooks_path}"
+        cache_path "/var/opt/supermarket/local-mode-cache"
```

Comment:
@robbkidd Yeah I would place the cache in its own directory (I think `/var/opt/supermarket/cache` is least surprising). You also want to run a `supermarket-ctl reconfigure` if you want to see a number of the chef-zero stuff get created.